package.json

```diff
@@ -80,29 +82,29 @@
     "eslint-plugin-react": "^7.24.0",
     "expect": "^27.0.6",
     "jest": "^27.0.6",
+    "jest-puppeteer": "^5.0.4",
     "loader-utils": "^2.0.0",
     "lodash": "^4.17.21",
     "mini-css-extract-plugin": "^2.1.0",
-    "mocha": "^8.4.0",
     "nock": "^13.1.1",
     "nodemon-webpack-plugin": "^4.5.2",
     "pegjs": "^0.10.0",
     "pegjs-loader": "^0.5.6",
     "pino-pretty": "^5.1.1",
+    "puppeteer": "^9.1.1",
+    "request-promise-native": "^1.0.9",
```

Comment:
this is a deprecated package, we're hoping to remove. I assume it was added to resolve failing [app.test](https://github.com/alphagov/paas-admin/blob/main/src/components/app/app.test.ts#L5)

removing that line would require a rewrite of those tests

Reply:
I removed the dependency. The refectory of `app.test` should be addressed outside of this story.